docs/infrastructure/eks-quickstart.md

```diff
@@ -1,204 +1,159 @@
+    Note that, as stated in the documentation, if your `.aws/credentials` are fairly complex you might need to set `AWS_SDK_LOAD_CONFIG=1` for Terraform to work correctly:
 
-In order to setup the infrastructure you'll need recent versions of:
+    ```bash
+    export AWS_SDK_LOAD_CONFIG=1
+    ```
 
--   [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
--   [terraform >= v0.12](https://learn.hashicorp.com/terraform/getting-started/install.html)
--   [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
--   [helm](https://helm.sh/docs/using_helm/#installing-helm)
--   Credentials with access to docker hub (you should receive this with your license)
+1. Clone the Chart repository and change to the directory that includes the Terraform configuration files provided by Codacy:
 
-Please follow the documentation in the above links to setup these tools for your OS.
-They can usually be easily installed using your package manager. For instance, on
-macOS, if you are using [Homebrew](https://brew.sh), you can just run:
+    ```bash
+    git clone https://github.com/codacy/chart.git
+    cd chart/docs/infrastructure/EKS/
+    ```
 
-```bash
-brew install awscli terraform kubernetes-helm kubectl
-```
+    The folder includes the following infrastructure stacks:
 
-You'll also need to setup the CLI credentials for your AWS account. See how
-to do it the [AWS](https://docs.aws.amazon.com/polly/latest/dg/setup-aws-cli.html)
-and [Terraform](https://www.terraform.io/docs/providers/aws/index.html) documentation.
-Note that, as stated in the documentation, if your `.aws/credentials` are fairly
-complex you might need to set `AWS_SDK_LOAD_CONFIG=1` for Terraform to work correctly.
+    * **backend**: Optional S3 bucket for storing the Terraform state and the DynamoDB table for state locking
+    * **main**: Amazon EKS cluster, including the setup of all network and node infrastructure to go from zero to a fully functional cluster
+    * **setup**: Additional setup to be performed before installing Codacy on your vanilla Amazon EKS cluster
 
-## TL;DR
+## 2. Set up the Terraform state storage backend
 
-### Setting up an EKS cluster for Codacy
+The [backend](https://www.terraform.io/docs/backends/index.html) stores the current and historical state of your infrastructure.
 
-Assuming **AWS is well configured** and you fulfill all the prerequisites,
-the setup without state storage, can be done with:
+Although using the backend is optional, we recommend that you deploy it, particularly if you are planning to use these templates to make modifications to the cluster in the future:
 
-```bash
-$ export AWS_SDK_LOAD_CONFIG=1
-$ cd main/
-$ terraform init && terraform apply # should take around 15 mins
-$ cd ../setup/
-$ terraform init && terraform apply
-$ aws eks update-kubeconfig --name codacy-cluster
-```
+1. Initialize Terraform and apply the changes inside the `backend/` directory, then follow Terraform's instructions:
```

Comment:
I don't think `apply the changes inside the` reflects what's being done. Maybe `deploy the infrastructure described in the` would be better suited.

Reply:
Great feedback, as I'm not familiarized with the Terraform "lingo". :+1: